ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -683,6 +683,8 @@ export const ConceptHomepage = fp.flow(withCurrentWorkspace(), withCurrentConcep
         currentInputString, currentSearchString, domainInfoError, domainsLoading, inputErrors, loadingDomains, surveyInfoError,
         standardConceptsOnly, showSearchError, searching, selectedDomain, selectedSurvey, selectedConceptDomainMap, selectedSurveyQuestions,
         surveyAddModalOpen, surveysLoading} = this.state;
+      const conceptDomainCards = conceptDomainList.filter(domain => domain.domain !== Domain.PHYSICALMEASUREMENT);
```

Comment:
nit: Maybe rename the const to coneptDomainSurveyCards? Just something to make sure it not contains PhysicalMeasurement

Reply:
This is not the survey list, just all the domains except physical measurements. I moved these here so I wouldn't have to filter `conceptDomainList` multiple times in `render()`. `conceptSurveysList` doesn't need to be filtered, so I call it directly in `render()`.